app/controllers/users_controller.rb

```diff
@@ -1,4 +1,27 @@
 class UsersController < ApplicationController
+  def show
+    @user = User.find params[:id]
+  end
   def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new user_params
+    if @user.save
+      flash[:success] = t"users_controller.wl"
```

Comment:
Thiếu khoảng trắng, em sửa lại như này nhé:
```suggestion
      flash[:success] = t "users_controller.wl"
```